nevergrad/functions/base.py

```diff
@@ -65,7 +65,12 @@ def __new__(cls: tp.Type[EF], *args: tp.Any, **kwargs: tp.Any) -> EF:
         inst._descriptors["function_class"] = cls.__name__
         return inst  # type: ignore
 
-    def __init__(self: EF, function: tp.Callable[..., tp.Loss], parametrization: p.Parameter) -> None:
+    def __init__(
+        self: EF,
+        function: tp.Callable[..., tp.Loss],
+        parametrization: p.Parameter,
+        special_evaluation_function: tp.Optional[tp.Callable[..., tp.Loss]] = None,
```

Comment:
`evaluation_function` is already the way to provide a special evaluation function, so in my point of view this is a duplicate, except to shortcut the need of subclassing. But I am not sure that it is that much more convenient, so I would like to avoid this extra parameter.

let's discuss this further offline, ad I'll try to provide another way